### PR TITLE
added Dto models, BookController, missing methods in repository and service

### PR DESCRIPTION
**Pull request by maxymmusiienko**

### CODE REVIEW

File: src/main/java/com/example/bookshop/dto/BookDto.java [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+
+    private String author;
+
+    private String isbn;
+
+    private BigDecimal price;
+
+    private String description;
+
+    private String coverImage;
```
Comment (okuzan) [inline src/main/java/com/example/bookshop/dto/BookDto.java]: remove empty lines between the fields

File: src/main/java/com/example/bookshop/dto/BookDto.java [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+    }
+
+    public Long getId() {
+        return id;
+    }
+
+    public void setId(Long id) {
+        this.id = id;
+    }
+
```
Comment (okuzan) [inline src/main/java/com/example/bookshop/dto/BookDto.java]: use lombok annotations instead, like @Data

File: src/main/java/com/example/bookshop/model/Book.java [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+                + '\''
+                + ", price="
+                + price
+                + ", description='"
+                + description
+                + '\''
+                + ", coverImage='"
+                + coverImage
+                + '\''
+                + "} ";
```
Comment (okuzan) [inline src/main/java/com/example/bookshop/model/Book.java]: takes too much space

File: src/main/java/com/example/bookshop/repository/BookRepositoryImpl.java [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+    @Override
+    public List<Book> findAll() {
+        Session session = null;
+
+        try {
+            session = sessionFactory.openSession();
+            Query<Book> query = session.createQuery("FROM Book", Book.class);
+            return query.list();
+        } catch (Exception e) {
+            throw new RuntimeException("Error while getting all users", e);
```
Comment (okuzan) [inline src/main/java/com/example/bookshop/repository/BookRepositoryImpl.java]:
```suggestion
            throw new RuntimeException("Error while getting all the books", e);
```

**Review by okuzan — CHANGES_REQUESTED** [review]

**Review by kozhukhovsky — APPROVED** [review]

**Pull request closed without merging**